packages/suite-web/e2e/tests/analytics/events.test.ts

```diff
@@ -134,6 +135,7 @@ describe('Analytics Events', () => {
         // nothing should be reported until analytics is initialized and enabled
         cy.wrap(requests).should('have.length', 0);
 
+        cy.disableFirmwareHashCheck();
```

Comment:
Maybe it would be better to just disable it at the start rather than in the middle of the test? Unless it's necessary for the test, I think it kinda breaks the flow of arrange/act/assert (or given/when/then). 

If that is possible, then I think we can move the prefixedVisit and this straight to the beforeEach.

Reply:
Oh yes, good call! At first I placed it at beginning, but the test was hanging because of this, and moving the procedure fixed it, but I'm really puzzled why :confused: 
[This is the run](https://github.com/trezor/trezor-suite/actions/runs/11479515554/job/31948847103), and here is artifact 
![Analytics Events -- reports suite-ready after enabling analytics on app initial run (failed)](https://github.com/user-attachments/assets/2ac2f7ac-8e62-4797-9127-b00f15f6fa1f)

Reply:
Oh yeah, see comment in 9b8bd8bdd2d014b3d78dc61d1ff0d0e79e5dec86
I am not a fan of using `cy.wait` to randomly fix things, but in this case I think I proved the reason why exactly it's necessary.

Reply:
I see. It makes more sense like this anyway, thanks.